game.pro

```diff
@@ -9,20 +9,22 @@ include(game.pri)
 include(game_view.pri)
 
 # Use the C++ version that all team members can use
-CONFIG += c++11
-QMAKE_CXXFLAGS += -std=c++11
+CONFIG += c++17
+QMAKE_CXXFLAGS += -std=c++17
 

```

Comment:
This is scary, i.e. to expect all team members have C++17. I guess someone else put it in, so I am going to accept this.

Reply:
Found the culprit https://github.com/tresinformal/game/pull/548/commits/69c8740f1e8e1aa1ada5f3816f11030453ffec46 ;)